src/lib/eslint/LWCEslintStrategy.ts

```diff
@@ -69,17 +77,31 @@ export class LWCEslintStrategy implements EslintStrategy {
 		return EslintStrategyHelper.getDefaultConfig(this.recommendedConfig, ruleName);
 	}
 
+	getRuleMap(): Map<string, ESRule> {
+		const unfilteredRules: Map<string,ESRule> = EslintStrategyHelper.getBaseEslintRules();
+
+		// Add all LWC-specific rules to the map.
+		for (const key of Object.keys(rules)) {
+			// NOTE: LWC rules have no `type` attribute by default. We're going to default them to `problem` for now.
+			const typedRule = {...rules[key]};
+			typedRule.meta.type = 'problem';
```

Comment:
LWC rules have no type, so we default to `problem`.

Reply:
I like the idea of defaulting type values when missing but wondering if we may cause confusion for people who've used eslint outside of the plugin. What do you think about using a different name to denote categories without type?

Reply:
I'm not sure what that value would be, though. "None" and "Default" don't really make sense as category names. What do you think?